ur_robot_driver/include/ur_robot_driver/hardware_interface.hpp

```diff
@@ -188,6 +188,7 @@ class URPositionHardwareInterface : public hardware_interface::SystemInterface
 
   bool robot_program_running_;
   bool non_blocking_read_;
+  double robot_program_running_copy_;
```

Comment:
Why do we need this “copy_” value?

Reply:
Because the state interface only can take double values, so it is the boolean variable represented as a double